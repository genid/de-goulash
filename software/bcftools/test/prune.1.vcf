##fileformat=VCFv4.2
##reference=file://some/path/human_g1k_v37.fasta
##contig=<ID=1,length=2147483647>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	s1	s2	s3
1	101	.	T	A	.	.	AF=0.3	GT	0/1	0/1	0/1
1	102	.	T	A	.	.	AF=0.2	GT	0/1	0/1	0/1
1	103	.	T	A	.	.	AF=0.1	GT	0/1	0/0	0/0
1	104	.	T	A	.	.	AF=0.3	GT	0/0	0/0	0/0
1	105	.	T	A	.	.	AF=0.2	GT	0/0	0/0	0/0
1	106	.	T	A	.	.	AF=0.1	GT	0/1	1/1	1/1
1	107	.	T	A	.	.	AF=0.3	GT	0/1	0/0	0/1
1	108	.	T	A	.	.	AF=0.2	GT	0/1	1/1	0/1
