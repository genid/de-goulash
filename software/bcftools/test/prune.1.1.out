##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##reference=file://some/path/human_g1k_v37.fasta
##contig=<ID=1,length=2147483647>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
##INFO=<ID=R2,Number=1,Type=Float,Description="Pairwise r2 with the POS_R2 site">
##INFO=<ID=POS_R2,Number=1,Type=Integer,Description="The position of the site for which R2 was calculated">
##INFO=<ID=LD,Number=1,Type=Float,Description="Pairwise Lewontin's D' (PMID:19433632) with the POS_LD site">
##INFO=<ID=POS_LD,Number=1,Type=Integer,Description="The position of the site for which LD was calculated">
##INFO=<ID=HD,Number=1,Type=Float,Description="Pairwise Ragsdale's \hat{D} (PMID:31697386) with the POS_HD site">
##INFO=<ID=POS_HD,Number=1,Type=Integer,Description="The position of the site for which HD was calculated">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	s1	s2	s3
1	101	.	T	A	.	.	AF=0.3	GT	0/1	0/1	0/1
1	102	.	T	A	.	.	AF=0.2;POS_R2=101;POS_LD=101;POS_HD=101;R2=1;LD=1;HD=0	GT	0/1	0/1	0/1
1	103	.	T	A	.	.	AF=0.1;POS_R2=102;POS_LD=102;POS_HD=102;R2=0.105235;LD=0.145076;HD=0	GT	0/1	0/0	0/0
1	104	.	T	A	.	.	AF=0.3;POS_R2=103;POS_LD=103;POS_HD=103;R2=0.105235;LD=0;HD=0	GT	0/0	0/0	0/0
1	105	.	T	A	.	.	AF=0.2;POS_R2=104;POS_LD=104;POS_HD=104;R2=1;LD=0;HD=0	GT	0/0	0/0	0/0
1	106	.	T	A	.	.	AF=0.1;POS_R2=105;POS_LD=105;POS_HD=105;R2=0.755358;LD=0;HD=0	GT	0/1	1/1	1/1
1	107	.	T	A	.	.	AF=0.3;POS_R2=106;POS_LD=106;POS_HD=106;R2=0.25;LD=0.790569;HD=-0.0208333	GT	0/1	0/0	0/1
1	108	.	T	A	.	.	AF=0.2;POS_R2=107;POS_LD=107;POS_HD=107;R2=1;LD=1;HD=-0.0416667	GT	0/1	1/1	0/1
